publish_leap151_jeos.config

```diff
@@ -0,0 +1,61 @@
+# vim:syntax=sh
+
+leap_version=15.0
```

Comment:
15.1?

Reply:
I think GH doesn't send notifications for marking something as resolved, so I'll do it: Resolved.